tensorflow/lite/micro/kernels/svdf.h

```diff
@@ -47,6 +48,8 @@ extern const int kSvdfInputActivationStateTensor;
 extern const int kSvdfOutputTensor;
 
 // TensorflowLite Micro-specific reference implementation for Integer SVDF.
+// One version for each of int8_t and int16_t time_weights/state.
+template <typename T>
 void EvalIntegerSvdfReference(TfLiteContext* context, TfLiteNode* node,
```

Comment:
move template definition to the header: https://google.github.io/styleguide/cppguide.html#Self_contained_Headers

In this case, that would be really annoying so we could have two functions

`EvalInt8SvdfReference` and `EvalInt16SvdfReference` in the header. And these two can be implemented with a call to the function template in the .cc file

Reply:
Done.